src/App.jsx

```diff
@@ -97,7 +97,9 @@ class App extends Component {
                 if (won) {
                   this.setState({ points: points + 1, timer: newTimer + 3000 });
                 } else if (points > 0) {
```

Comment:
Although this is an old change I must ask what the intent is.

It might not be possible, but if I were to have `0.5` points, a failed answer would result in `-0.5` points. So this would not really guard against negative points